tests/test-lz4-list.py

```diff
@@ -0,0 +1,282 @@
+#! /usr/bin/env python3
```

Comment:
@gabrielstedman : something I have recently noticed while running this test script locally : 
it seems to fail, at least there are messages on the screen that looks like errors.

```
test_list_14M-lz4f-2f--content-size.lz4(2/2)
     Frame           Type Block Checksum           Compressed         Uncompressed     Ratio
         1       LZ4Frame   B7I    XXH32              5800117             11534336     50.29%
         2       LZ4Frame   B7I    XXH32              1585234              3145728     50.39%


test_block (__main__.TestVerbose) ... ERROR
test_checksum (__main__.TestVerbose) ... ERROR
test_compressed (__main__.TestVerbose) ... ERROR
test_filename (__main__.TestVerbose) ... ok
test_frame_number (__main__.TestVerbose) ... ok
test_frame_type (__main__.TestVerbose) ... ERROR
test_ratio (__main__.TestVerbose) ... ERROR
test_uncompressed (__main__.TestVerbose) ... ERROR

======================================================================
ERROR: test_block (__main__.TestVerbose)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test-lz4-list.py", line 155, in test_block
    if "--BD" in self.cvinfo.file_frame_map[i]:
IndexError: list index out of range
```

Yet the test itself doesn't signal any failure through the use of a return error code,
so CI tests have remained silent about it, hence this situation is not presumed recent.

I realize that it's unclear if this test is _supposed to fail_,
and if not, there could be 2 issues : 
- why is the test failing, should the `lz4` CLI be fixed to comply to those tests
- how to make the script correctly report the test error

Reply:
I noticed the exit code is not being set properly I already have an easy fix for this here:
https://github.com/gabrielstedman/lz4/commit/c9d2c7197710d0f532a786a02daa5ba994438de2

But I can't reproduce this failure as the test succeeds for me in the `dev` branch, could you please let me know how did you reproduce this failure?